packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -66,7 +66,8 @@ export class BoosterGraphQLDispatcher {
       try {
         this.logger.debug(`Decoding current user from auth token: ${envelope.token}`)
         envelope.currentUser = await this.boosterTokenVerifier.verify(envelope.token)
-      } catch (e) {
+      } catch (err) {
+        const e = err as string
```

Comment:
🤔  Is this intended? Even while you can throw a string, I think that the language wraps it in an Error object...

Reply:
not now! now errors are `unkown` and you are forced to define the type. In that particular case the error was a string.

Reply:
That's weird, maybe we should try to fix that and make sure errors are always wrapped in an `Error` object 🤔

Reply:
Fixed!! I checked and the function throwing the error was generating a normal `Error` so simply was a matter of changing the streaming error from simply pass the error to `err.message`